gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/bottomsheet/GravatarQuickEditorBottomSheet.kt

```diff
@@ -47,20 +47,21 @@ public fun GravatarQuickEditorBottomSheet(
     modalBottomSheetState: SheetState = rememberModalBottomSheetState(skipPartiallyExpanded = true),
 ) {
     val coroutineScope = rememberCoroutineScope()
-    val screenHeightDp = LocalConfiguration.current.screenHeightDp
 
     GravatarTheme {
         ModalBottomSheet(
-            modifier = Modifier
-                .heightIn(min = DEFAULT_PAGE_HEIGHT, max = screenHeightDp.dp * 0.9f),
             onDismissRequest = { onDismiss(GravatarQuickEditorDismissReason.Finished) },
             sheetState = modalBottomSheetState,
             dragHandle = { QEDragHandle() },
+            containerColor = MaterialTheme.colorScheme.surface,
+            tonalElevation = 1.dp,
```

Comment:
The `ModalBottomSheet` has some changes in the latest 1.3.0-rc01 version that make our bottom sheet look weird. These changes fix that. I decided not to update the library and keep those changes to make sure the QE will look properly if third-party apps have newer dependencies.